### PR TITLE
Remove require bits

### PR DESCRIPTION
**Pull request by Biglr**
Removes all imports of the Bits library from the script itself since the script runner a) provides them inline and b) the require functionality will be discontinued.
@J2thatsme make sure all custom device types on v2 are updated accordingly as well. It will lead to runtime errors of the scripts if left unchanged.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,2756 +1,8 @@
 {
   "name": "device-types",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```
Comment (mariusgiger) [inline package-lock.json:4]: please upgrade node and run npm install again ;)

Comment (Biglr) [reply package-lock.json:4]: I just installed both, nodejs has version 12.18.2, npm is version 6.14.8 :thinking:

Comment (mariusgiger) [reply package-lock.json:4]: it should be v15.12.0 and 7.6.3

Comment (Biglr) [reply package-lock.json]: well, npm is not 7.7.4 :man_shrugging:

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**